src/model/quality.rs

```diff
@@ -0,0 +1,49 @@
+use combine::{attempt, choice, optional, parser::char::string, Parser, Stream};
+use strum_macros::{Display, EnumIter};
+use Quality::*;
+
+#[derive(Clone, Copy, Debug, Display, EnumIter, Eq, PartialEq)]
+pub enum Quality {
+    #[strum(serialize = "")]
+    Major,
```

Comment:
strum can implement `Display` for you. By default it'll use the case names, but I'm overriding that here for custom output.